assets/src/scripts/components/graph/state/layout.js

```diff
@@ -130,13 +130,20 @@ export const getAxisYBBox = memoize(opts => createSelector(
  * @param  {Object} state   Redux state
  * @return {Object}         {x, y, width, height} of viewBox
  */
-export const getViewBox = memoize(opts => createSelector(
+export const getViewBox = memoize((opts) => createSelector(
     getContainerSize(opts),
     getAxisYBBox(opts),
     (containerSize, axisYBBox) => {
-        const aspectRatio = containerSize.height / containerSize.width || 0;
+        let aspectRatio;
+        if (opts.graphType === 'water-levels') {
+            aspectRatio = containerSize.height / containerSize.width || 0;
+        } else if (opts.graphType === 'construction') {
+            // add an arbitrary number to the aspect ratio to keep it long and not wide
```

Comment:
This little '+1' is where 90% of the magic lies, but I couldn't figure out a way to do things better. The normal 'aspect ratio' keeps the water levels graph short and wide. Adding a magic number to that value, makes it taller and thinner. The number '1' just happens to work really well. I tried separating the logic for the water levels graph and the well diagram, but that lead to a huge amount of 99% redundant code. In the end this seemed like a really hacky work around.

Reply:
After testing the application with the additional media break points, it seemed that this could be eliminated and still have an acceptable appearance.  So, I took it out. One less bit of magic to worry about.